mart/utils/utils.py

```diff
@@ -263,12 +263,12 @@ def get_resume_checkpoint(config: DictConfig) -> Tuple[DictConfig, str]:
 
         resume_checkpoint = resume_checkpoint[0]
         log.info(f"Resuming from {resume_checkpoint}")
+        # Save the ckpt_path in cfg for fit() and test().
+        overrides += [f"+ckpt_path={resume_checkpoint}"]
```

Comment:
Why is this necessary?

Reply:
We allow `--resume=[folder]` but `fit()` and `test()` requires a path to the `ckpt` file.

Reply:
This changes the logic of what happened before though.

Notice that before the logic enabled the user to specify `ckpt_path`. Now you override any user control. See the logic difference?

Reply:
FYI I think there should be a test for this case.

Reply:
I don't think it changes the program behavior. `ckpt_path` was not used because we have a more advanced `--resume` feature.

Reply:
Okay if that is the case then why not just use `cfg.set("ckpt_path", resume_checkpoint)` instead of using overrides? Is there a good reason to use the overrides functionality?